lib/devices/cpu_device_manager.js

```diff
@@ -51,12 +51,15 @@ export default class CpuDeviceManager {
 
 class CpuDevice {
   constructor(id, active=false) {
-    this.id = id
+    this.id = id;
```

Comment:
> I actually am thinking of writing a docker config scheme validator
As a JSON schema? you publish somewhere, that would be awesome...
Because then it could be used in different languages and applications...